src/main/java/org/cactoos/collection/Sliced.java

```diff
@@ -51,23 +50,11 @@ public Sliced(final int start, final int count, final T... items) {
      * Ctor.
      * @param start Starting index
      * @param count Maximum number of elements for resulted iterator
-     * @param iterable Source iterable
+     * @param src Source iterable
      */
-    public Sliced(final int start, final int count,
-        final Iterable<T> iterable) {
-        this(start, count, new CollectionOf<T>(iterable));
-    }
-
-    /**
-     * Ctor.
-     * @param start Starting index
-     * @param count Maximum number of elements for resulted iterator
-     * @param collection Source collection
-     */
-    public Sliced(final int start, final int count,
-        final Collection<T> collection) {
-        super(() -> new CollectionOf<T>(
-            new org.cactoos.iterable.Sliced<T>(start, count, collection)
+    public Sliced(final int start, final int count, final Iterable<T> src) {
+        super(new Sticky<>(
```

Comment:
@victornoel please, use the same indentation here which you used bellow (IMHO it's better).